site/content/docs/5.0/forms/file.md

```diff
@@ -6,16 +6,12 @@ group: forms
 toc: true
 ---
 
-{{< callout info >}}
-The recommended plugin to animate custom file inputs is [bs-custom-file-input](https://www.npmjs.com/package/bs-custom-file-input); it's what we use here in our docs.
-{{< /callout >}}
-
 ## Default
 
 The file input is the most gnarly of the bunch and requires additional JavaScript if you'd like to hook them up with functional *Choose file...* and selected file name text.
```

Comment:
This might need to be reworded since it won't require extra JS, but our plugin :)

Reply:
if you have any ideas do not hesitate 😅

Reply:
Also, not sure if this should be listed as a component or not (probably not)

@mdo please have a look and let us know :)